tests/in/glsl/expressions.frag

```diff
@@ -74,7 +74,11 @@ void testArrayConstructor() {
 	float tree[1] = float[1](0.0);
 }
 
+float global;
+void privatePointer(inout float a) {}
+
 out vec4 o_color;
 void main() {
+	privatePointer(global);
```

Comment:
nit: something happened with indentation

Reply:
Sorry was using tabs and didn't notice